lib/system/setops.nim

```diff
@@ -9,6 +9,17 @@ func incl*[T](x: var set[T], y: T) {.magic: "Incl".} =
     a.incl(4)
     assert a == {1, 2, 3, 4, 5}
 
+func toSet*[char|byte|bool|int16|uint16|enum](ar: openArray[T]): set[T] =
```

Comment:
we should avoid adding new symbols to `system.nim` unless properly justified.

=> move this to `std/setutils.nim` which can grow more builtin-set specific procs
(follows family of std/enumutils, etc)